shift-react-app/src/steps.ts

```diff
@@ -0,0 +1,65 @@
+// tslint:disable:no-console
+import {
+  readFileSync,
+  writeFileSync,
+} from 'fs';
+import { join } from 'path';
+import findLastIndex from 'lodash.findlastindex';
+import { spawn } from 'child_process';
+
+export function setupProxy() {
+  const proxyFile = join('src', 'setupProxy.js');
+  const proxyCode = `const { setupProxy } = require('@binaris/shift-local-proxy');
+module.exports = setupProxy(__dirname);
+`;
+  writeFileSync(proxyFile, proxyCode, { flag: 'wx' });
+  console.log(`Created ${proxyFile}, please commit this file`);
+}
+
+export function installPackages() {
+  return new Promise((resolve, reject) => {
+    const dependencies = [
+      '@binaris/shift-local-proxy',
+      '@binaris/shift-babel-macro',
+      '@binaris/shift-fetch-runtime',
+      '@binaris/shift-db',
+    ];
+    const args = [
+      'install',
+      '--save',
+      '--loglevel',
+      'error',
+    ].concat(dependencies);
+    const child = spawn('npm', args, { stdio: 'inherit' });
```

Comment:
We should use yarn if user created react app with yarn preference.

Reply:
There should at least be an issue tracking this.

Reply:
created #104 despite this never being a requirement